DolbyIO/Source/DolbyIOModule/Public/DolbyIO.h

```diff
@@ -0,0 +1,181 @@
+// Copyright 2022 Dolby Laboratories
+
+#pragma once
+
+#include "DolbyIO/SdkEventObserver.h"
+#include "DolbyIOParticipantInfo.h"
+#include "Subsystems/GameInstanceSubsystem.h"
+
+#include "Engine/EngineTypes.h"
+#include "HAL/ThreadSafeBool.h"
+
+#include "DolbyIO.generated.h"
+
+namespace DolbyIO
+{
+	class FAuthenticator;
+	class FSdkAccess;
+}
+
+/** The Dolby.io Virtual World plugin game instance subsystem. */
+UCLASS(Abstract, Blueprintable)
+class DOLBYIOMODULE_API UDolbyIO : public UGameInstanceSubsystem, public DolbyIO::ISdkEventObserver
+{
+	GENERATED_BODY()
+
+public:
+	/*
+	 *
+	 * Functions for controlling the plugin's behavior, callable from Blueprints.
+	 *
+	 */
+
+	/** Initializes or refreshes the client access token. Initializes the plugin unless already initialized. Triggers
+	 * OnInitialized if initialization was successful.
+	 * @param Token - Client access token.
+	 */
+	UFUNCTION(BlueprintCallable, Category = "Dolby")
```

Comment:
Should we name the category `Dolby.io`? Dolby also has an Unreal plugin and we probably don't want to mix the two (just in case).

Reply:
I agree. It would also be cleaner since everywhere else we refer to Dolby.io or DolbyIO, but in this Category there is just Dolby.

Reply:
FYI, the Unity plugin is using `DolbyIO`

Reply:
What's the decision then? Should Unity rename to Dolby.io or should Unreal rename to DolbyIO?

Reply:
@4thlabs whats your opinion for Unity?

Reply:
Namespace is dolbyio in C++ and in C#, so I've base my naming on it to be consistent, DolbyIO is the category I've used.

Reply:
I must add, that of course it is a very technical way of thinking, but I find it also good in term of understanding.

Reply:
I think "Dolby.io" is better when it's user-facing, though. In this case, users can see the string "Dolby.io" in Unreal Editor, which in my opinion is better than "DolbyIO". I think we should be using "Dolby.io" wherever possible.

Reply:
I see Unity will change to "Dolby.io Comms" and Unreal shall follow. Will be addressed fully when we resolve differences between Unity and Unreal.